docs/config.yaml

```diff
@@ -208,6 +208,9 @@ clouds:
       # OPTIONAL: boolean: whether or not to set the image/snapshot as public on
       # initial upload (AMIs of released builds are always made public)
       public: true
+    # OPTIONAL: list of streams to build and replicate AWS Windows License Included images for
+    winli:
+      - "4.19-9.6"
```

Comment:
Is there a better way to implement this? AIUI, our stream approach might not make much sense anymore in the RHCOS case.